examples/cordova-sample/MyApp/www/js/index.js

```diff
@@ -55,13 +55,6 @@ const app = {
     Purchases.setEmail("garfield@revenuecat.com");
   },
 
-  setupShouldPurchasePromoProductListener: function() {
-    Purchases.addShouldPurchasePromoProductListener((makeDeferredPurchase) => {
-      console.log("This codes executes right before making the purchase");
-      makeDeferredPurchase();
-      console.log("This codes executes right after making the purchase");
-    });
-  },
```

Comment:
I have absolutely no memory of why I removed this

Reply:
probably safe to revert

Reply:
Oh, I was wondering the same, and it's because it's duplicated as a function in line 283, so it's safe to go